invenio_jsonschemas/errors.py

```diff
@@ -25,7 +25,7 @@ def __init__(self, schema, *args, **kwargs):
         """
         self.schema = schema
         super(JSONSchemaNotFound, self).__init__(
-            'Schema "{}" not found'.format(schema), *args, **kwargs
+            "Schema {schema} not found".format(schema=schema), *args, **kwargs
```

Comment:
minor: a little f-string to simplify things?

 ```suggestion
            f"Schema {schema} not found", *args, **kwargs
```

Reply:
I thought about that, but there is no difference and I tried to modify as less as possible the code.